example/ding-dong-bot.ts

```diff
@@ -65,8 +65,7 @@ bot
     const room = m.room()
     console.log((room ? '[' + room.topic() + ']' : '')
                 + '<' + m.from().name() + '>'
-                + ':' + m.toStringDigest()
-    )
+                + ':' + m.toStringDigest())
```

Comment:
Please keep the original coding style.
Just add a comma to the end, and keep `)` at the new line.